routes/items/item_response.js

```diff
@@ -16,25 +16,25 @@ const allItemsQuery = user_id => (
   { order: [['createdAt', 'ASC']], where: { user_id }, FETCH_ATTRIBUTES }
 )
 
-const filteredItemsQuery = (Item, query, user_id) => ({ items, tree }) => {
-  const where = Object.assign( {}, whereSearch( query ), whereCompleted( query), { user_id } )
+const filterClause = ( query, user_id ) => {
+  const where = Object.assign(
+    {},
+    whereSearch( query ),
+    whereCompleted( query ),
+    whereStarred( query ),
+    { user_id }
+  )
 
-  if( Object.keys( where ).length === 1 ) {
-    return { items, tree }
-  } else {
-    return Item.findAll({ where, attributes: [ 'id' ] })
-      .then( pruneTree( items, tree ))
-  }
+  return { where }
```

Comment:
Sin e we're just declaring a variable to return it on the next line, can we consolidate this to a single statement, as above?